Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN uname -a &&\
     CGO_ENABLED=${CGO_ENABLED} GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
     make build && make cel-key
 
-FROM docker.io/alpine:3.19.1
+FROM docker.io/alpine:3.20.0
```

Comment:
See 1st line. Looks like dependabot isn't smart enough to find `docker pull golang:1.22.3-alpine3.20`